crates/core/src/sql/parser.rs

```diff
@@ -0,0 +1,38 @@
+use crate::db::datastore::locking_tx_datastore::MutTxId;
+use crate::db::relational_db::RelationalDB;
+use crate::sql::ast::SchemaViewer;
+use spacetimedb_expr::check::parse_and_type_sub;
+use spacetimedb_expr::errors::TypingError;
+use spacetimedb_expr::expr::RelExpr;
+use spacetimedb_expr::ty::TyCtx;
+use spacetimedb_lib::db::raw_def::v9::RawRowLevelSecurityDefV9;
+use spacetimedb_lib::identity::AuthCtx;
+use spacetimedb_schema::schema::RowLevelSecuritySchema;
+
+pub struct RowLevelExpr {
+    pub sql: RelExpr,
+    pub def: RowLevelSecuritySchema,
+}
```

Comment:
Do we need a new type? We don't use the `RelExpr` at all, other than getting the table id from it.

Reply:
I expect we need this when later we execute the `rls` policy